package.json

```diff
@@ -37,7 +37,7 @@
     "karma-webpack": "^1.7.0",
     "mocha": "^2.4.5",
     "raw-loader": "^0.5.1",
-    "react": "^0.14.8",
+    "react": ">=0.14.8 <15.2.0",
```

Comment:
Btw you can amend the commit?

Don't need to create a whole new pr!

Sent from my iPhone

> On 27 Jul 2016, at 21:53, Develliot <notifications@github.com> wrote:
> 
> In package.json:
> 
> > @@ -37,7 +37,7 @@
> >      "karma-webpack": "^1.7.0",
> >      "mocha": "^2.4.5",
> >      "raw-loader": "^0.5.1",
> > -    "react": "^0.14.8",
> > +    "react": ">=0.14.8 <15.2.0",
> No worries I only did this because I couldn't guarantee compatibility after 15.1 I'll make a new pull request.
> …
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>